### ISSUE TITLE
3.7.0 breaks system notifications. 

### ISSUE THREAD

**Issue by cru-Luis-Rodriguez** [Adobe-Consulting-Services/acs-aem-commons#946]
Hello, 

I upgraded ACS commons from 3.4.0 to latest 3.7.0 on an AEM 6.2 instance. Now the notification feature does not work and throws an exception. I downgraded to 3.6.2 and the feature works on that version. Are there plans to fix this in 3.7.x next release? 

```
Error during include of component '/apps/acs-commons/components/utilities/system-notifications'

Error Message:

java.lang.ExceptionInInitializerError
Processing Info:

Page	=	/etc/acs-commons/notifications/test	
Resource Path	=	/etc/acs-commons/notifications/test/jcr:content	
Cell	=	system-notifications	
Cell Search Path	=	system-notifications	
Component Path	=	/apps/acs-commons/components/utilities/system-notifications	
Sling Request Progress:

```

[Full exception ](https://gist.github.com/cru-Luis-Rodriguez/e398029b0ba3ce7ccb9e6300595b2c82)

**Comment by davidjgonzalez** [comment]
@cru-Luis-Rodriguez  now there is!

**Comment by cru-Luis-Rodriguez** [comment]
@davidjgonzalez 

Thank you!

-Luis

**Comment by davidjgonzalez** [comment]
@cru-Luis-Rodriguez  if you have a chance could you try this snapshot out?

[acs-aem-commons-content-3.7.1-SNAPSHOT-min.zip](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/883564/acs-aem-commons-content-3.7.1-SNAPSHOT-min.zip)
[acs-aem-commons-content-3.7.1-SNAPSHOT.zip](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/883563/acs-aem-commons-content-3.7.1-SNAPSHOT.zip)

**Comment by cru-Luis-Rodriguez** [comment]
That did the trick 👍 

Thanks
![screen shot 2017-03-30 at 3 45 44 pm](https://cloud.githubusercontent.com/assets/6875635/24523135/0748a3a4-1560-11e7-945d-ec386b26576b.png)

**Comment by justinedelson** [comment]
@davidjgonzalez has this been fixed in the feature/6.2-compatibility branch?

**Comment by davidjgonzalez** [comment]
@justinedelson #954 fixes this; travis is checking it out as we speak.
